README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+I would say semantic HTML is more than just the way a page looks visually. There are links and containers and working functions. Not just color and style. 
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+Block sets elements vertically in a columnand inline sets elements horizontally in a row
```

Comment:
block doesn't set elements vertically.  inline doesn't set things horizontally.  

Your homework is to read this article.  Then I'll quiz you afterwards.  https://coursework.vschool.io/html-block-vs-inline/